seismic_hazard_analysis/gms/gcim_emp.py

```diff
@@ -1,23 +1,24 @@
-"""Contains functions for computation of the empirically GMM (i.e. parametric) based GCIM"""
+"""
+Contains functions for computation of the empirically GMM (i.e. parametric) based GCIM
+"""
 
-from typing import Dict, Sequence, Union, Tuple
+from typing import Dict, Sequence, Tuple, Union
 
-import pandas as pd
 import numpy as np
-from scipy import stats
+import pandas as pd
+import scipy as sp
 
-from sha_calc import disagg
-from sha_calc import ground_motion as gm
+from .. import disagg, hazard, im_correlations
```

Comment:
Not sure if these kinda imports follow our guides or not, If I remember single dot was ok?

Reply:
Hmh I don't think we defined anything in regards to relative imports? I always use relative imports inside the package, find it much clearer since I usually populate the __init__.py files to make usage of the package easier.

Reply:
two dots for stuff in the parent folder.. I agree with @claudio525 and am happy with sensible use of relative path